server/Controllers/contestControllers.js

```diff
@@ -28,6 +28,29 @@ async function registerContest(req, res) {
   }
 }
 
+async function updateContest(req, res) {
```

Comment:
is there a possibility to update array of participants?
and change current class property isFinished from false to true?